src/elements/ProjectsMonitoringCounters/JobsCounters.js

```diff
@@ -0,0 +1,184 @@
+          link: () => navigate('/projects/jobsMonitoring/jobs/completed'),
+          statusClass: 'completed'
+        }
+      ]
+    }),
+    [jobs, navigate]
+  )
+
+  const workflowsStats = useMemo(
+    () => ({
+      all: {
+        counter: workflows.length,
+        link: () => navigate('/projects/jobsMonitoring/workflows/all')
+      },
+      counters: [
+        {
+          counter: workflows.filter(workflow => workflow.state?.value === 'running').length,
+          link: () => navigate('/projects/jobsMonitoring/workflows/running'),
+          statusClass: 'running'
+        },
+        {
+          counter: workflows.filter(workflow => ['error', 'failed'].includes(workflow.state?.value))
+            .length,
+          link: () => navigate('/projects/jobsMonitoring/workflows/failed'),
+          statusClass: 'failed'
+        },
+        {
+          counter: workflows.filter(workflow =>
+            ['completed', 'succeeded'].includes(workflow.state?.value)
+          ).length,
+          link: () => navigate('/projects/jobsMonitoring/workflows/completed'),
+          statusClass: 'completed'
+        }
+      ]
+    }),
+    [workflows, navigate]
+  )
+
+  return (
+    <StatsCard className='stats'>
+      <StatsCard.Header title='Jobs and Workflows'>
+        <DatePicker
+          date={filter.dates.value[0]}
+          dateTo={filter.dates.value[1]}
+          selectedOptionId={PAST_24_HOUR_DATE_OPTION}
+          label=''
+          onChange={handleDateSelection}
+          type='date-range-time'
+          withLabels
+        />
+      </StatsCard.Header>
+      <StatsCard.Body>
+        <StatsCard.Col>
+          <h6 className='stats__subtitle'>Jobs</h6>
+          <span className='stats__counter-display'>
+            {jobsLoading ? <Loader section small secondary /> : jobStats.all.counter}
+          </span>
+          <ul className='stats__counters'>
+            {jobStats.counters.map(counter => (
+              <li className='link' onClick={counter.link} key={`${counter.statusClass}-jobs`}>
+                {jobsLoading ? <Loader section small secondary /> : counter.counter}
+                <i className={`state-${counter.statusClass}-job`}></i>
+              </li>
+            ))}
+          </ul>
+        </StatsCard.Col>
+        <StatsCard.Col>
+          <h6 className='stats__subtitle'>Workflows</h6>
+          <span className='stats__counter-display'>
+            {workflowsLoading ? <Loader section small secondary /> : workflowsStats.all.counter}
+          </span>
+          <ul className='stats__counters'>
+            {workflowsStats.counters.map(counter => (
+              <li className='link' onClick={counter.link} key={`${counter.statusClass}-jobs`}>
+                {workflowsLoading ? <Loader section small secondary /> : counter.counter}
+                <i className={`state-${counter.statusClass}-job`}></i>
+              </li>
+            ))}
+          </ul>
+        </StatsCard.Col>
```

Comment:
Looks like the same template is used twice. You can create a function for generating this template